src/styling-panel-definition.js

```diff
@@ -1,4 +1,86 @@
-const getStylingPanelDefinition = (bkgOptionsEnabled) => ({
+import styleDefaults from './style-defaults';
+import propertyResolver from './utils/property-resolver';
+import { fontFamilyOptions, colorOptions } from './utils/style-utils';
+
+export const getStyleEditorDefinition = () => ({
+  items: {
+    labelSection: {
+      component: 'panel-section',
+      translation: 'Common.Label',
+      items: {
+        labelItem: {
+          items: {
+            fontFamily: {
+              component: 'dropdown',
+              ref: 'style.font.fontFamily',
+              options: fontFamilyOptions,
+              defaultValue: styleDefaults.FONT_FAMILY,
+            },
+            fontWrapper: {
+              component: 'inline-wrapper',
+              type: 'items',
+              grouped: true,
+              items: {
+                fontStyle: {
+                  component: 'font-style-buttons',
+                  width: false,
+                  ref: 'style.font.style',
+                  defaultValue: ['bold'],
+                  matchValueFormat: true,
```

Comment:
I might change this to `formatValue` instead.
In the new styling button component (font-style-buttons), the value is an object when in the old version it is in an array. Using these in different places requires a converter that converts the values to the right format. Whenever style button group is used this property `matchValueFormat` should be set to true.

Reply:
Actually I think we should should use the existing `type` option.
So add `type: 'array'` here instead